app/views/idv/doc_auth/_notices.html.erb

```diff
@@ -1,6 +0,0 @@
-<% if flow_session[:notice].present? %>
```

Comment:
To consider separately: Aside from this, there's only one other reference to `flow_session[:notice]`, and it's in a partial that doesn't appear to be included anywhere:

https://github.com/18F/identity-idp/blob/master/app/views/idv/doc_auth/_document_capture_notices.html.erb

Seems like we could remove that partial and all logic which assigns `flow_session[:notice]`?

Reply:
Good catch, just pulled all that out.